storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/entities/Person.java

```diff
@@ -0,0 +1,68 @@
+package com.pushtorefresh.storio.sample.db.entities;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.pushtorefresh.storio.sample.db.tables.PersonsTable;
+import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteColumn;
+import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@StorIOSQLiteType(table = PersonsTable.TABLE)
+public class Person {
```

Comment:
Annotations will only trigger generation of Put, Get and Delete Operation resolvers in compile time, since you don't use autogenerated code for `Person` class — you don't need annotations :)